src/dodal/beamlines/training_rig.py

```diff
@@ -25,6 +26,7 @@
 #
 
 BL = get_beamline_name("p47")
```

Comment:
nit: can this be changed to `p46`

Reply:
v. good point, if p46 is going to be our default